site-src/guides/implementers.md

```diff
@@ -355,3 +355,13 @@ or both.
 Implementations that support cross-namespace references MUST watch ReferenceGrant
 and reconcile any ReferenceGrant that points to an object that's referred to by
 an in-scope Gateway API object.
+
+## Backend Protocol
+
+Not all implementations of Gateway API support automatic protocol selection. In some cases protocols are disabled without an explicit opt-in.
```

Comment:
Nit: It would be good to reword this to focus on the context that this is the Implementer's Guide, and thus intended for implementers to read. What do implementers need to know about this feature? I think the focus in this guide should be to talk about how you might implement it, what this concept usually maps to in common underlying dataplanes like Envoy or NGINX, and where implementers can find more information (I think you already have that part in "References", but it might be good to have more specifics such as the well defined protocols that we recommend implementations support. It would also be good to include keywords such as websockets and HTTP/2 for implementers that may be looking for how to support these concepts with their Gateway API implementation.